src/lib/undux/plugins/logger.js

```diff
@@ -7,10 +7,13 @@ import type { State } from '../GDStore'
 export const log = logger.child({ from: 'undux' })
 
 const withPinoLogger: Effects<State> = store => {
-  //disabled  currently causes pino-redact to crash, in any case we need this just for debugging
-  // store.onAll().subscribe(({ key, previousValue, value }) => {
-  //   log.info('changed', key, 'from', previousValue, 'to', value)
-  // })
+  store.onAll().subscribe(({ key, previousValue, value }) => {
+    log.info('store content changed', {
```

Comment:
is that all ? is it works everywhere without crashes ?
have you checked what we're keeping in the store ?

according to the state shapes (in src/lib/undux/SimpleStore or GDStore) we're keeping large objects/collections such as feeds. Also we're keeping wallet/userStorage (could remove from logging changes just here e.g. `if key != 'wallet' log.info(` )

Reply:
@yaroslav-fedyshyn-nordwhale i removed censorship for everything except plain object

se let's just skip logging changes for objects such as wallet or userStorage here
just check them by `key` argument and don't log everything or log some value (true/false or null/'<present>') depending of the key set or empty